sw/device/tests/BUILD

```diff
@@ -2284,11 +2284,15 @@ opentitan_functest(
     cw310 = cw310_params(
         bitstream = "//hw/bitstream:rom_otp_test_unlocked0",
         tags = ["jtag"],
-        test_cmds = OPENTITANTOOL_OPENOCD_TEST_CMDS,
+        test_cmds = [
```

Comment:
This rule should be named `openocd_test` (but github won't allow me to comment on line 2282).